### ISSUE TITLE
passing parameters to dask-ssh

### ISSUE THREAD

**Issue by simone-codeluppi** [dask/distributed#1354]
Hi 
it will be great to be able to start multiple workers with `dask-ssh` and passing setting parameters such as `--memory-limit` or `--nprocs`. By doing that it will be possible to set up a cluster pretty much automatically.

**Comment by mrocklin** [comment]
Yes, this would be great.   Do you have any interest in helping by submitting a PR for this?  The relevant code is in https://github.com/dask/distributed/blob/master/distributed/deploy/ssh.py and https://github.com/dask/distributed/blob/master/distributed/cli/dask_ssh.py
